apps/studio/components/interfaces/Support/SupportForm.tsx

```diff
@@ -384,6 +384,13 @@ const SupportForm = ({ setSentCategory }: SupportFormProps) => {
                 ) : (
                   <></>
                 )}
+                {(values.severity === 'Urgent' || values.severity === 'High') && (
+                  <p className="text-sm text-foreground-light mt-2">
+                    We do our best to respond to everyone as quickly as possible; however, we
+                    prioritize things based on production status. We ask that you please reserve
+                    High and Urgent severity for production-impacting issues only.
```

Comment:
just a thought - wondering if "however, prioritization will be based on production status..." might be better

'we prioritize things' sounds a lil informal